extensions-core/lookups-cached-global/src/main/java/io/druid/server/lookup/namespace/URIExtractionNamespaceCacheFactory.java

```diff
@@ -134,17 +129,15 @@ public String call() throws Exception
                 {
                   final String version = puller.getVersion(uri);
                   try {
-                    long lastModified = Long.parseLong(version);
-                    if (lastModified <= lastCached) {
-                      final DateTimeFormatter fmt = ISODateTimeFormat.dateTime();
+                    if (version.equals(lastVersion)) {
```

Comment:
Yes. `version.equals(lastVersion)` should return false if `lastVersion` is `null`. Added a comment clarifying this.